aman/src/components/Label.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Label({ lableTitle }) {
+  return (
+    <div>
+      <label htmlFor="" className="text-sm font-normal">
+        {lableTitle} <span className="text-red-600 ml-1">*</span>
```

Comment:
Instead of adding red * in label, try to make it a separate component which will appear with label only if boolean condition is true. This is because label will not be always required at all places.

Reply:
done